src/core/mac/sub_mac.hpp

```diff
@@ -551,7 +553,8 @@ class SubMac : public InstanceLocator, private NonCopyable
 #if OPENTHREAD_CONFIG_PLATFORM_USEC_TIMER_ENABLE
     static constexpr uint32_t kEnergyScanRssiSampleInterval = 128; // RSSI sample interval for energy scan, 128 usec
 #else
-    static constexpr uint32_t kEnergyScanRssiSampleInterval = 1; // RSSI sample interval during energy scan, 1 msec
+    static constexpr uint32_t kEnergyScanRssiSampleInterval =
+        1000; // RSSI sample interval during energy scan, 1000 usec
```

Comment:
Making comment shorter so it fits in same line?
```suggestion
    static constexpr uint32_t kEnergyScanRssiSampleInterval = 1000; // RSSI sample interval for energy scan, in usec
```